git_theta/metadata.py

```diff
@@ -108,7 +93,7 @@ def from_metadata_dict(cls, d):
         return cls(metadata)
 
     @classmethod
-    @file_or_name
+    @file_or_name(file="r")
```

Comment:
Yeah, haven't looked into why this was needed too deeply, but I'm pretty sure since `cls` is the first argument `file_or_name` opens that instead of `file`.

```
In [8]: class A:
   ...:     @classmethod
   ...:     @file_or_name(f='w')
   ...:     def method(cls, f):
   ...:         print(cls, type(cls))
   ...:         print(f, type(f))
   ...:         f.write('abcd')
```

Reply:
Thanks! Adding the arguments solves it here. I'll (eventually :grimacing:) get it fixed upstream lol.